src/particle.cpp

```diff
@@ -277,7 +277,9 @@ void Particle::event_cross_surface()
       boundary().lattice_translation[1] != 0 ||
       boundary().lattice_translation[2] != 0) {
     // Particle crosses lattice boundary
-    cross_lattice(*this, boundary());
+
+    bool verbose = settings::verbosity >= 10 || trace();
+    cross_lattice(*this, boundary(), verbose);
```

Comment:
Would it be easy to move the particle trace object into `Geometron` from the particle, so as to avoid having to carry this variable through the call stack, or would that be more trouble than it's worth?

Reply:
Hm, I was hoping to keep the `Geometron` as minimal as possible. Did you want to use `trace()` in random ray? If so I say let's put it in.

Reply:
This seems reasonable. If we end up wanting to use trace() in random ray (or another particle type) down the line, it can always be moved then. For now, I think the extra argument works well.